app.js

```diff
@@ -8,6 +8,10 @@ var app = express();
 var mongoose = require('mongoose');
 mongoose.connect(databaseConfig.getMongoURI());
 
+/* Models */
+var user = require('./model/user')(mongoose);
```

Comment:
I think this will eventually be a UserDAO. Doing active record is a little weird. Right now, this module has a dependency on mongoose which it uses to setup the schema, some test data (e.g. Bob), and return a function to find a user by name.